src/fping.c

```diff
@@ -1323,6 +1332,15 @@ void finish()
     if (stats_flag)
         print_global_stats();
 
+    if (reachable_flag) {
+        if ((num_hosts-num_unreachable >= reachable)) {
+            printf(">=%d hosts are reachable\n", reachable);
```

Comment:
Should you maybe also include the number of reachable hosts in the output?

Reply:
I have printed the number of reachable hosts instead of the given statement.